app/assets/stylesheets/categories.scss

```diff
@@ -0,0 +1,4 @@
+.grey-row {
+  color: #7a7a7a;
+  background-color: #e3e3e3;
+}
```

Comment:
FYI and future reference, bulma also provides these:
https://bulma.io/documentation/helpers/color-helpers/

Reply:
thanks! they're a bit more heavy-handed (or colors) than we were going for. i actually want to lighten that grey even more now that i'm looking at it again! :)

Reply:
Hmm, a bit surprised their range doesn't cover it. IIRC, they had like a dozen shades of gray!

Reply:
duh. you're right. they can be mixed and matched!